documentation/tech-stack.md

```diff
@@ -0,0 +1,26 @@
+|-----------|---------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| FE        | React + UI libraries such as: Ant design, bootstrap, material UI, bulma, tailwind CSS | - Client first choice - Widely supported and documented - Facebook-backed                            | - A bit harder to learn than Vue                                                                   |
+| FE        | Vue                                                                                   | - JP has experience - Easy learning curve                                                            | - Not as much support and documentation - No major company backing                                 |
+| FE        | Angular                                                                               | - Most mature of all three frameworks                                                                | - Steep learning curve                                                                             |
+| BE        | Node with Express.JS                                                                  | - Client first choice - Uses JS, like the front-end - JP has experience - V8 engine (very fast)      | - Callback syntax can become complex  - Slower than Python if trying to do heavy computation tasks |
+| BE        | Python with one of Django, Flask, Nameko                                              | - Python syntax (subjective advantage)                                                               | - Doesn't natively support asynchronous development - Slower than node                             |
+| DB        | Firebase                                                                              | - Client first choice - JP has experience - Easy to learn and use - Large community                  | - Expensive when scaling                                                                           |
+| DB        | AWS                                                                                   | - Large suite of tools (authentication, data processing) - More capable & customizable than Firebase | - Steeper learning curve - Harder to set up                                                        |
+| DB        | MongoDB                                                                               | - Harder to set up than AWS                                                                          | - More predictable pricing                                                                         |
```

Comment:
Currently when I view the file, it looks like the statements for MongoDB's pros and cons might be reversed. I think "more predictable pricing" is a pro, and "harder to set up than AWS" is a con.

Reply:
Agree